enterprise-suite/console-api/default-monitors.json

```diff
@@ -83,7 +83,7 @@
           "filters": {
             "quantile": "0.5"
           },
-          "period": "15m",
+          "period": "5m",
```

Comment:
Is this intentional for this PR?

Reply:
Yes, I forgot to mention it in PR description. This is one of the changes I made in console-api repo, more extensive akka monitors tweaks will follow.